_translator-files/po/de/Include-Server-Commands.po

```diff
@@ -3,128 +3,129 @@
 
 #. type: Plain text
 #: ../wiki/en/Include-Server-Commands.md:6
 #, no-wrap
 msgid "`-F`  or `--fastupdate`     Reduces latency if Clients connect with \"Enable Small Network Buffers\" option. Requires faster CPU to avoid dropouts, and more bandwidth to enabled Clients.\n"
-msgstr ""
+msgstr "`-F` oder `--fastupdate` Verringert die Latenz, wenn Clients mit der Option \"Aktiviere kleine Netzwerkpuffer\" verbunden sind. Erfordert eine schnelle CPU, um Aussetzer zu vermeiden, und mehr Bandbreite für aktivierte Clients.\n"
 
 #. type: Plain text
 #: ../wiki/en/Include-Server-Commands.md:7
 #, no-wrap
 msgid "`-l`  or `--log`            Enable logging, set path and file name                                                                    \n"
-msgstr ""
+msgstr "`-l` oder `--log` Protokollierung einschalten, Pfad und Dateiname festlegen\n"
 
 #. type: Plain text
 #: ../wiki/en/Include-Server-Commands.md:8
 #, no-wrap
 msgid "`-L`  or `--licence`        Show an agreement window before users can connect\n"
-msgstr ""
+msgstr "`-L`  oder --licence` Zeigt ein Zustimmungsfenster an, bevor Benutzer eine Verbindung herstellen können\n"
 
 #. type: Plain text
 #: ../wiki/en/Include-Server-Commands.md:9
 #, no-wrap
 msgid "`-m`  or `--htmlstatus`     Enable HTML status file, set path and file name\n"
-msgstr ""
+msgstr "`-m` oder `--htmlstatus` HTML-Statusdatei aktivieren, Pfad und Dateiname festlegen\n"
 
 #. type: Plain text
 #: ../wiki/en/Include-Server-Commands.md:10
 #, no-wrap
 msgid "`-o`  or `--serverinfo`     Location details in the format:  `[name];[city];[locale value]` (see [values](https://doc.qt.io/qt-5/qlocale.html#Country-enum)) Registered Servers only\n"
-msgstr ""
+msgstr "`-o` oder `--serverinfo` Ortsangaben im Format:  `[Name];[Stadt];[Gebietsschema-Wert]` (siehe [Werte](https://doc.qt.io/qt-5/qlocale.html#Country-enum)) Nur registrierte Server\n"
 
 #. type: Plain text
 #: ../wiki/en/Include-Server-Commands.md:11
 #, no-wrap
 msgid "`-P`  or `--delaypan`       Start with delay panning enabled See [notes](#delay-panning)\n"
-msgstr ""
+msgstr "`-P` oder `--delaypan` Start mit aktiviertem Delay-Panning Siehe [Notizen](#delay-panning---stereo-effekt)\n"
 
 #. type: Plain text
 #: ../wiki/en/Include-Server-Commands.md:12
 #, no-wrap
 msgid "`-R`  or `--recording`      Include a writeable path where the files should be stored (in quotes if needed).  See [Options](#options).  \n"
-msgstr ""
+msgstr "`-R` oder `--recording` Gib einen beschreibbaren Pfad ein, in dem die Dateien gespeichert werden sollen (in Anführungszeichen, falls erforderlich). Siehe [Optionen](#optionen).  \n"
 
 #. type: Plain text
 #: ../wiki/en/Include-Server-Commands.md:13
 #, no-wrap
 msgid "`--norecord`       Disable recording when enabled by default by `-R`                                                 \n"
-msgstr ""
+msgstr "`--norecord` Aufnahme deaktivieren. Durch Angabe von `-R` wird die Aufnahme standardmäßig aktiviert\n"
```

Comment:
@gilgongo this needs to be rephrased. I think it's not well written. What it does is basically making recording available but it needs to be enabled via JSONRPC or SIGUSR

Reply:
Not for me to do :-) @pljones  is best to consult I think.